tests/review-child-handoff.test.sh

```
@@ -848,7 +1039,9 @@ assert len(serialized)==limit+1
 with open(path,'w',encoding='utf-8') as stream:
     json.dump(value,stream,separators=(',',':'))
 PY
-! _uberdev_child_prepare review_pr.review.correctness "$MAX_PLUS_ONE_HANDOFF" "$UBERDEV_CHILD_RESULT" "$UBERDEV_CHILD_STATUS" dispatch >/dev/null 2>&1
+MAX_PLUS_ONE_SHA256="$(file_sha256 "$MAX_PLUS_ONE_HANDOFF")"
+! _uberdev_child_prepare review_pr.review.correctness "$MAX_PLUS_ONE_HANDOFF" \
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)